python/pyspark/sql/catalog.py

```diff
@@ -164,6 +169,65 @@ def listTables(self, dbName: Optional[str] = None) -> List[Table]:
             )
         return tables
 
+    def getTable(self, tableName: str, dbName: Optional[str] = None) -> Table:
```

Comment:
can we remove the `dbName` parameter? I think we should also deprecate table related APIs with db and table name parametes at scala side, and promote the single string parameter APIs. cc @amaliujia

Reply:
I think it's a good idea.
functions with `dbName` and `tableName` are now somewhat confusing, when `tableName` start to support 3L namespace.

Let me update this PR.